flight1handler.go

```diff
@@ -62,7 +62,16 @@ func flight1Generate(c flightConn, state *State, cache *handshakeCache, cfg *han
 			RenegotiatedConnection: 0,
 		},
 	}
-	if cfg.localPSKCallback == nil {
```

Comment:
Is it possible to use PSK suites without `localPSKCallback` being set?

Reply:
This is not about PSK suites. This setup extension is used for elliptic curves. A good example of this is the cipher [AECDH-AES128-SHA](https://ciphersuite.info/cs/TLS_ECDH_anon_WITH_AES_128_CBC_SHA), which I have used as a custom cipher to implement the [OCF specification](https://openconnectivity.org/specs/OCF_Security_Specification_v2.2.5.pdf).